charts/common/values.yaml

```diff
@@ -32,6 +32,9 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+secret: {}
```

Comment:
I would add a small header comment here explaining what this section does (and maybe a small warning about putting clear text passwords in your helm values).